crates/polars-plan/src/logical_plan/builder_dsl.rs

```diff
@@ -190,7 +117,7 @@ impl DslBuilder {
                     use_statistics,
                 },
                 cloud_options,
-                metadata,
```

Comment:
https://github.com/pola-rs/polars/issues/21851

In `scan_parquet` metadata wasn't always `None`